src/backend/base/pyproject.toml

```diff
@@ -63,6 +63,7 @@ cryptography = "^42.0.5"
 asyncer = "^0.0.5"
 pyperclip = "^1.8.2"
 uncurl = "^0.0.11"
+cassio = "^0.1.7"
```

Comment:
I think all you gotta do is move this over to the root of pyproject.toml and it should be good to go. 

Thank you!

Reply:
Awesome, thanks @ogabrielluiz. Should be good for another look!